### ISSUE TITLE
Feature request: Better job progress indication

### ISSUE THREAD

**Issue by pedjas** [LaserWeb/LaserWeb4#591]
LaserWeb is not great in showing job progress. Practically all we have is queue downcounter which only shows number of commands left to execute.

LaserWeb has good Simulator. It could be used to show job progress too. And that would be ideal as it would show visually how job progresses.

Also some estimation of total time for job to finish would be good to have.

**Comment by cprezzi** [comment]
I agree, but I don't have time to implement something like that. Unfortunately it's low on the priority list.
